docs/demo.md

```diff
@@ -0,0 +1,314 @@
+# Demo Show
+
+In this demo, we will show how to create a distributed traning in Jupyter Notebook with the help of Ciao.
+
+[Video on Vimeo](https://vimeo.com/288833803)
+
+The code in the cell is:
+
+```python
```

Comment:
note for future improvement

The user experience here is that they essentially have to copy/paste their already well-written code and submit to kubeflow, i.e. mostly the training part.  What I have in mind is that at last, we provide a more smoother experience such that it's possible to perform "develop-train-eval".